lib/nylas/model.rb

```diff
@@ -18,22 +18,27 @@ def self.included(model)
     def save
       raise_if_read_only
       result = if id
-                 api.execute(method: :put, payload: attributes.serialize, path: resource_path)
+                 exec
```

Comment:
Hmm, did you mean to leave `exec` here?

Reply:
ah, good catch! It's a no-op since it doesn't do anything, but leaving a system call lying out is not a good thing!